src/solver/expressions/visitors/EvalVisitor.cpp

```diff
@@ -21,6 +21,7 @@
 
 #include "antares/solver/expressions/visitors/EvalVisitor.h"
 
+#include <cmath>
```

Comment:
`#include <cmath>` : seems useless (removing it still allows a successful compilation)

Reply:
done